### PR TITLE
PL Release Notes 788xx

### PR DESCRIPTION
**Pull request by rashminandasahoo**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/harness-platform.md [hunk 1]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 23, 2023, version 788xx
+
+### What's new
+
+- You can now add service accounts to your resource group. (PL-31867)
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:24]: You can now add specific service accounts to your resource groups

File: release-notes/harness-platform.md [hunk 2]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+  By doing this, you can prevent accidental or deliberate misuse of API keys by restricting who can generate them from what service accounts.
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:26]: from which service accounts

File: release-notes/early-access.md [hunk 3]
```diff
@@ -12,6 +12,13 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 23, 2023, version 788xx
+
+### Harness Platform
+
+- By enabling the feature flag `PL_NEW_SCIM_STANDARDS`, any CRUD operation on a user now returns user group details. (PL-31496)
```
Comment (KapilDigani) [inline release-notes/early-access.md:19]: now returns the details of the user groups that user is part of

File: release-notes/harness-platform.md [hunk 4]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+- A failed decryption of secrets managed by the Harness Secret Manager causes the secret value inside values.yaml to be resolved as null. (PL-32043)
+  
+  The pipeline execution now fails with an exception if there is a failure in decrypting secrets.
+
+- Despite having an active license, the CD module is not visible. (PLG-2047)
+  
+  A code enhancement has fixed this issue.
+
+- SMTP configurations with special characters in the SMTP configuration name throw an `Invalid request` error. The SMTP configuration name is also used as the SMTP secret and secrets should not have any special characters. (PL-31774, ZD-40679)
```
Comment (KapilDigani) [inline release-notes/harness-platform.md:46]: A little confusing - please consider revising the test.

**Review by SudarshanPurohit — APPROVED** [review]

File: release-notes/early-access.md [hunk 5]
```diff
@@ -12,6 +12,13 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
 Additionally, the release notes below are only for NextGen SaaS. FirstGen SaaS release notes are available [here](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes) and Self-Managed Enterprise Edition release notes are available [here](/release-notes/self-managed-enterprise-edition).
 :::
 
+## March 23, 2023, version 788xx
+
+### Harness Platform
+
+- By enabling the feature flag `PL_NEW_SCIM_STANDARDS`, any CRUD operation on a user now returns the details of the user groups that user is part of. (PL-31496)
```
Comment (roshnisarangadharan) [inline release-notes/early-access.md:19]:
```suggestion
- By enabling the feature flag,`PL_NEW_SCIM_STANDARDS`, any CRUD operation on a user now returns the details of the user groups that the user is part of. (PL-31496)
```

File: release-notes/harness-platform.md [hunk 6]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+- You can now add specific service accounts to your resource group. (PL-31867)
+  
+  By doing this, you can prevent accidental or deliberate misuse of API keys by restricting who can generate them from which service accounts.
+
+- You can now enter usernames as a comma separated string while adding users in **Users(name or email)**. (PL-29630)
+  
+### Early access
+
+- By enabling the feature flag `PL_NEW_SCIM_STANDARDS`, any CRUD operation on a user now returns the details of the user groups that user is part of. (PL-31496)
```
Comment (roshnisarangadharan) [inline release-notes/harness-platform.md:32]:
```suggestion
- By enabling the feature flag, `PL_NEW_SCIM_STANDARDS`, any CRUD operation on a user now returns the details of the user groups that the user is part of. (PL-31496)
```

File: release-notes/harness-platform.md [hunk 7]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+  
+  The pipeline execution now fails with an exception if there is a failure in decrypting secrets.
+
+- Despite having an active license, the CD module is not visible. (PLG-2047)
+  
+  A code enhancement has fixed this issue.
+
+- SMTP configurations with special characters in the SMTP configuration name throw an `Invalid request` error. This happens because the SMTP configuration name is used to construct the secret name and secret names should not have any special characters. (PL-31774, ZD-40679)
+
+  This issue has been fixed by replacing special characters in SMTP configuration names with `-` before creating secret.
```
Comment (roshnisarangadharan) [inline release-notes/harness-platform.md:48]: "..before creating a secret" or "..before creating secrets"

File: release-notes/harness-platform.md [hunk 8]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+- User invites throw an `Invalid request` error when 2FA is enabled in the Account scope. (PL-31276)
+ 
+    A code enhancement has fixed this issue.
```
Comment (roshnisarangadharan) [inline release-notes/harness-platform.md:52]: There is extra space in the beginning of the line.

**Review by roshnisarangadharan — DISMISSED** [review]
approved with some suggestions

File: release-notes/harness-platform.md [hunk 9]
```diff
@@ -17,6 +17,54 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+  You can use this to verify what groups a given user belongs to.
+
+### Fixed issues
+
+- A failed decryption of secrets managed by the Harness Secret Manager causes the secret value inside values.yaml to be resolved as null. (PL-32043)
+  
+  The pipeline execution now fails with an exception if there is a failure in decrypting secrets.
+
+- Despite having an active license, the CD module is not visible. (PLG-2047)
```
[the remark this answers is not in the document]
Comment (rashminandasahoo) [reply release-notes/harness-platform.md:42]: I guess the logic is handled by the PL team :)

**Review by roshnisarangadharan — DISMISSED** [review]

**Pull request merged**